client/src/panels/result_graphs/results_common.js

```diff
@@ -116,41 +116,36 @@ const type_by_data_type = {
 };
 
 const indicator_target_text = (indicator) => {
-  const {
-    target_type,
-    target_min, 
-    target_max,
-    target_narrative,
-    measure,
-  } = indicator;
   const target_unspecified_display = text_maker("unspecified_target");
 
   const measure_display = (measure) => !_.isEmpty(measure) && `( ${measure} )`;
 
-  const get_display_case = (data_type, min, max, narrative, meas) => {
+  const get_display_case = ({target_type, target_min, target_max, target_narrative, measure, seeking_to}) => {
     switch(target_type){
       case 'num':
       case 'num_range':
       case 'dollar':
       case 'dollar_range':
       case 'percent':
       case 'percent_range': {
-        if ( /range/.test(target_type) && (min && max) ){
-          return `${text_maker("result_range_text")} ${formats[type_by_data_type[data_type]](+min)} ${text_maker("and")} ${formats[type_by_data_type[data_type]](+max)}` + (measure_display(meas) || "");
-        } else if (min && max && min === max){
-          return `${text_maker("result_exact_text")} ${formats[type_by_data_type[data_type]](+min)}` + (measure_display(meas) || "");
-        } else if (min && !max){
-          return `${text_maker("result_lower_target_text")} ${formats[type_by_data_type[data_type]](+min)}` + (measure_display(meas) || "");
-        } else if (!min && max){
-          return `${text_maker("result_upper_target_text")} ${formats[type_by_data_type[data_type]](+max)}` + (measure_display(meas) || "");
-        } else {
-          return target_unspecified_display; 
+        switch(seeking_to){
+          case 'target':
+            return `${text_maker("result_exact_text")} ${formats[type_by_data_type[target_type]](+target_min)}` + (measure_display(measure) || "");
+          case 'min':
+            return `${text_maker("result_upper_target_text")} ${formats[type_by_data_type[target_type]](+target_max)}` + (measure_display(measure) || "");
+          case 'max':
```

Comment:
I'm thinking of asking the pipeline to use clearer values for `seeking_to`, unless these are what Titan uses under the hood.

Maybe:
```
  target -> exact
  min -> minimize
  max -> maximize
  range -> in_range // or just leave as range
```

Reply:
Mainly it bugs me that the "min" case corresponds to use of the `target_max` value. Replacing minimize/maximize with more than/less than would probably be even better for clarity.

Reply:
These values are set in Titan, not going to change for now. 

Note Titan actually has three target value columns, min, max, and point. In the "target" seeking_to case, the pipeline stores the point value from Titan in both the min and max columns to save a column in the output.